server/jetstream_errors_generated.go

```diff
@@ -406,6 +409,15 @@ const (
 
 	// JSTemplateNameNotMatchSubjectErr template name in subject does not match request
 	JSTemplateNameNotMatchSubjectErr ErrorIdentifier = 10073
+
+	// JsConsumerDuplicateFilterSubjects consumer cannot have both FilterSubject and FilterSubjects specified
+	JsConsumerDuplicateFilterSubjects ErrorIdentifier = 10136
+
+	// JsConsumerMultipleFiltersNotAllowed consumer with multiple subject filters cannot use subject based API
+	JsConsumerMultipleFiltersNotAllowed ErrorIdentifier = 10137
+
+	// JsConsumerOverlappingSubjectFilters consumer subject filters cannot overlap
+	JsConsumerOverlappingSubjectFilters ErrorIdentifier = 10138
```

Comment:
Should this be like the others above to be consistent, like:
```suggestion
	JSConsumerOverlappingSubjectFilters ErrorIdentifier = 10138
```